data/org.freedesktop.portal.CaptureInput.xml

```diff
@@ -0,0 +1,406 @@
+              The capabilities available to this session. This is always a
+              subset of the requested capabilities.
+              See the SupportedCapabilities property for details. Note that
+              while a capability may be available to a session, there is no
+              guarantee a device with that capability is currently available
+              or if one does become available that it will trigger input capture.
+
+              It is best to view this set as a negative confirmation - a
+              capability that was requested but is missing is an indication that
+              this application may not capture events of that capability.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+    -->
+    <method name="CreateSession">
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="u" name="capabilities" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+
+    <!--
+        GetRegions:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @options: Vardict with optional further information
+        @regions: An array of 4 integers, each specifying a region's width, height, x/y offset and physical scale
+
+        Retrieve the available input regions for this session.
+        The regions may not be continuous and may be a logical representation
+        of the physical screens (e.g. a 4k screen may be represented as
+        low-resolution screen instead).
+
+        If the regions change (e.g. a monitor is unplugged), the
+        #org.freedesktop.portal.CaptureInput::RegionsChanged signal is emitted
+        and the application should re-request the current regions to update its internal state.
+
+        Note that regions are session-specific, there is no guarantee that two
+        applications see the same screen regions. An empty region list implies
+        that no pointer barriers can be set.
+
+        The physical scale by default is 1.0, or the proportion of
+        logical:physical size of the screen. This is required on systems that
+        do not have support for high-resolution displays but employ a mix of
+        high- and low-resolution displays.
+
+        For example, two 24 inch monitors, one 2k and one 4k are the same
+        physical size but the resolution on the 4k display is double
+        that of the 2k display. A cursor movement on the 2k screen covers N
+        pixels but that same physical cursor movement on the 4k display covers
+        2*N pixels. Likewise, a pointer barrier extending to the height of the
+        4k screen needs to be half as tall (in pixels) as that on the 2k screen
+        despite being the same physical size.
+    -->
+    <method name="GetRegions">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="a(uuiid)" name="regions" direction="out"/>
```

Comment:
One idea to add as an out argument is a `serial` that should be passed to `SetPointerBarriers` to make sure the used region source is the most recent one.

Reply:
done locally, will be in the next push